tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -648,6 +648,39 @@ class FooClass
                 null,
                 ['ensure_fully_multiline' => true],
             ],
+            'test function argument with multiline echo in it' => [
+            <<<'UNAFFECTED'
+<?php
+call_user_func(function ($arguments) {
+    echo 'a',
+      'b';
+}, $argv);
+UNAFFECTED
```

Comment:
No. `foo($a, $b);` is fine. But if you are using 
```
foo($a,
    $b);
```
By PSR2 or Symfony then you MUST write it as:
```
foo(
    $a,
    $b
);
```
See http://www.php-fig.org/psr/psr-2/#46-method-and-function-calls